bbc/simorgh
# NEWSWORLDSERVICE-1662: Update iChef image references to "ace-ichef"

Resolves JIRA NEWSWORLDSERVICE-1662

Overall changes
======
Replace references to the **/news/** iChef path to the new **/ace/standard/** wherever the buildIChefUrl function is used. Update tests, fixture data and snapshots.

Code changes
======

- Update buildiChefUrl function
- Update iChef URLs in tests snapshots, fixture data and stories that utilise the buildiChefUrl function.
- Refactor buildiChefUrl function to not add PNG parameter. Refactor byline extractor to remove redundent props.
- Note that there is a getBrandedImage function that still uses /news/.

Testing
======
- Pull down PR and search there are no instances of ichef.bbci.co.uk/news/ or ichef.test.bbci.co.uk/news/ (other than those related to the getBrandedImage function).
- Check through Storybook to ensure no broken images
- Check through local pages to ensure no broken images (including jpg, PNG, webp)
- On test - check Byline image renders as expected (unable to find a local example) - https://www.test.bbc.com/pidgin/articles/c39g87n9jp0o

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
